diff --git a/tritium/core/cxx_src/init_cuemol.cpp b/tritium/core/cxx_src/init_cuemol.cpp
--- a/tritium/core/cxx_src/init_cuemol.cpp
+++ b/tritium/core/cxx_src/init_cuemol.cpp
@@ -179,4 +179,4 @@
-#if defined(__APPLE__) || defined(_WIN32)
+#if defined(__APPLE__) || defined(_WIN32) || defined(__EMSCRIPTEN__)
     exports.Set(Napi::String::New(env, "bindPeer"),
                 Napi::Function::New(env, node_jsbr::bindPeer));
 #endif
diff --git a/tritium/core/cxx_src/services.hpp b/tritium/core/cxx_src/services.hpp
--- a/tritium/core/cxx_src/services.hpp
+++ b/tritium/core/cxx_src/services.hpp
@@ -12,3 +12,3 @@
-#if defined(__APPLE__) || defined(_WIN32)
+#if defined(__APPLE__) || defined(_WIN32) || defined(__EMSCRIPTEN__)
 Napi::Value bindPeer(const Napi::CallbackInfo &info);
 #endif
